deploy/install.sh

```diff
@@ -146,14 +171,28 @@ EOF
     -H "Content-Type: application/json; charset=utf-8" \
     -d "${json}")"
 
-  if echo "$response" | grep "${CONNECTION_NAME}" >/dev/null 2>&1; then
+  if echo "$response" | grep "${CLUSTER_NAME}" >/dev/null 2>&1; then
     echo "Kubernetes monitoring successfully setup."
   else
     echo "Error adding Kubernetes cluster to Dynatrace: $response"
   fi
 }
 
+checkForExistingCluster() {
+  response="$(curl -sS -X GET "${API_URL}/config/v1/kubernetes/credentials" \
+    -H "accept: application/json; charset=utf-8" \
+    -H "Authorization: Api-Token ${API_TOKEN}" \
+    -H "Content-Type: application/json; charset=utf-8")"
+
+  if echo "$response" | grep -Fqe "\"name\":\"${CLUSTER_NAME}\""; then
```

Comment:
I'm only accepting a customer argument to be used as a regex on this command because it's being validated before.